docs/attributes.md

```diff
@@ -12,14 +12,17 @@ Under the hood, attribute sources are a superset of functionality provided by [h
 
 Use `attribute` to extract the value of an attribute from markup.
 
-_Example_: Extract the `src` attribute from an image found in the block's markup.
+_Example_: Extract the `src` attribute from an image found in the block's markup. 
+`default` parameter can be used to set a default value. 
```

Comment:
I think we should move the `default` parameter to its own section because it's not specific to the "attribute" attribute.

Reply:
👍